### ISSUE TITLE
Calculating MPJPE

### ISSUE THREAD

**Issue by jamalknight** [karfly/learnable-triangulation-pytorch#98]
Hi there

I have run training and evaluation on this model (triangulation) and would like to calculate the error MPJPE based on the results this model has generated.

What is the process of calculating MPJPE based on the data calculated? Any info on the formula, and where to get the data (ground truth and estimated) would be helpful.

thanks

**Comment by jamalknight** [comment]
Am I correct in saying that the error rate (MPJPE) is written to the metric.json file located:

`/logs/human36m_vol_softmax_VolumetricTriangulationNet@16.07.2020-07:58:35/checkpoints/0056/metric.json`

thanks
